reana-ui/src/pages/profile/components/GitLabProjects.js

```diff
@@ -171,7 +171,9 @@ export default function GitLabProjects() {
                         verticalAlign="middle"
                       />
                       <List.Content>
-                        <List.Header as="a">{name}</List.Header>
+                        <List.Header as="a" href={url} target="_blank">
```

Comment:
We usually add `rel="noopener noreferrer"` to all the links, could you please add it also to the two links here? See for example:
https://github.com/reanahub/reana-ui/blob/d4036342dd5c2e89c840da15f993b112a8ef2fa6/reana-ui/src/pages/profile/components/GitLabProjects.js#L210-L214